.travis.yml

```diff
@@ -5,16 +5,33 @@ jdk:
 - openjdk10
 - openjdk11
 - openjdk12
+
+cache:
+  directories:
+    - $HOME/.m2
+
 script:
   - mvn clean install -B -V
+
 deploy:
-  provider: script
-  script: mvn -s .travis/deploy-settings.xml -DperformRelease -DskipTests -Dgpg.skip=true deploy
-  skip_cleanup: true
-  on:
-    repo: mvc-spec/mvc-spec
-    branch: master
-    jdk: oraclejdk8
+  - provider: script
+    script: mvn -s .travis/deploy-settings.xml -DperformRelease -DskipTests -Dgpg.skip=true deploy
+    skip_cleanup: true
+    on:
+      repo: mvc-spec/mvc-spec
+      branch: master
+      jdk: oraclejdk8
+  - provider: pages
+    skip_cleanup: true
+    script: skip
+    github_token: $GITHUB_TOKEN
+    keep_history: true
+    local_dir: spec/target/generated-docs
+    on:
+      repo: mvc-spec/mvc-spec
+      branch: master
+      jdk: oraclejdk8
```

Comment:
Shouldn't this be 'openjdk8'....?

Reply:
It should - copy-pasted the error I reported myself 🤦‍♂

Reply:
> But I would prefer to also use Netlify for the snapshots. This would be more consistent with the other deployment methods and creating GitHub tokens may get more complicated after migrating to EE4J because we won't have admin permissions for our repository anymore.

Fine for me - it's really just a matter of me being lazy 😋